src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -231,7 +231,8 @@ void DlgPrefLibrary::slotUpdate() {
     checkBox_SyncTrackMetadata->setChecked(
             m_pConfig->getValue(kSyncTrackMetadataConfigKey, false));
     checkBox_SeratoMetadataExport->setChecked(
-            m_pConfig->getValue(kSyncSeratoMetadataConfigKey, false));
+            m_pConfig->getValue(kSyncSeratoMetadataConfigKey, true));
```

Comment:
Same here, please revert.

Reply:
Ohh these were from my earlier commit(wrong one) forgot to revert it!